app/src/main/java/com/asfoundation/wallet/ui/iab/payments/common/model/WalletStatus.kt

```diff
@@ -0,0 +1,3 @@
+package com.asfoundation.wallet.ui.iab.payments.common.model
```

Comment:
Unfortunately it's not the same parameters but I'm up for suggestions so we can avoid repeated class names. The reason why I normally create new data objects instead of using Pair<> (or atleast I try to), is because Pair makes readability really bad. Despite the overhead I think it's worth it.